src/main/resources/application.yml

```
@@ -145,6 +145,8 @@ ide-sidecar:
     ccloud-project:
       client-id: 5c636508aa445d32c86f26b1
       eval-uri: https://confluent.cloud/ldapi/sdk/evalx/%s/contexts/%s
+  flink:
+    url-pattern: https://custom-flink.%s.%s.example.com
```

Is this right? Shouldn't it be this?

```suggestion
    url-pattern: https://flink.%s.%s.confluent.cloud
```